src/python/pants/backend/python/subsystems/setup.py

```diff
@@ -266,6 +267,31 @@ class PythonSetup(Subsystem):
             """
         ),
         advanced=True,
+        removal_version="2.15.0.dev0",
+        removal_hint=softwrap(
+            f"""
+            Pants will soon only support generating lockfiles via the Pex format, given the several
+            limitations of Poetry listed in this option.
```

Comment:
This might be read as insulting to Poetry. Maybe some softer phrasing such as "as Poetry is not a good a fit for Pants's needs."

Reply:
Thanks, good point! PTAL